src/plugins/discord/fetcher.js

```diff
@@ -131,8 +108,7 @@ export class Fetcher implements DiscordFetcher {
       mentions: (x.mentions || []).map((user) => user.id),
     }));
     const hasNextPage = results.length === messagesLimit;
-    const endCursor =
-      response.length > 0 ? response[response.length - 1].id : null;
+    const endCursor = response.length > 0 ? response[0].id : null;
```

Comment:
I didn't notice this when I reviewed this PR, oops. @BrianLitwin can you explain what this change is?

Reply:
The responses from the Discord Server (per our snapshots) come back in descending order (last message first, `5 4 3 2 1`).  I need to update the other methods that rely on endCursor for pagination; I made this change on messages because that was the method we used to test pagination with against the snapshots.